### PR TITLE
Added support for gtag

### PR DESCRIPTION
**Pull request by ThomasTNO**
Solves: https://github.com/readthedocs/sphinx_rtd_theme/issues/1128

Implements the new `gtag.js` Google Analytics.

### CODE REVIEW

**Comment by Blendify** [comment]
Is gtag backwards compatible with the old ga tracking code?

File: sphinx_rtd_theme/layout.html [hunk 1]
```diff
@@ -220,16 +220,23 @@
...
-
-    ga('create', '{{ theme_analytics_id }}', 'auto');
-    {%- if theme_analytics_anonymize_ip|tobool %}
-    ga('set', 'anonymizeIp', true);
-    {%- endif %}
-    ga('send', 'pageview');
+      (function () {
+        // New Google Site Tag (gtag.js) tagging/analytics framework
+        // https://developers.google.com/gtagjs
+        var script = document.createElement("script");
```
Comment (Blendify) [inline sphinx_rtd_theme/layout.html:226]: Why are you using JS to create the tag? Why not just create the tag yourself in html. This is a lot more complicated than what is recommended on https://developers.google.com/analytics/devguides/collection/upgrade/analyticsjs

**Comment by ThomasTNO** [comment]
> 
> 
> Is gtag backwards compatible with the old ga tracking code?

Yes, it is. https://developers.google.com/analytics/devguides/collection/upgrade/analyticsjs

File: sphinx_rtd_theme/layout.html (hunk 1 above)
Comment (ThomasTNO) [reply sphinx_rtd_theme/layout.html]: Fully agree, resulted from copy-paste laziness. Issue has been addressed

File: docs/configuring.rst [hunk 2]
```diff
@@ -97,7 +97,7 @@ Miscellaneous options
 .. confval:: analytics_id
 
     If specified, Google Analytics' javascript is included in your pages.
```
Comment (Blendify) [inline docs/configuring.rst:99]: We should add docs here about the particular implementation we use.

**Review by Blendify — COMMENTED** [review]
Changes seems fine, one small documentation comment.

@agjohnson is there anything that would cause concern here on read the docs?

File: docs/configuring.rst (hunk 2 above)
Comment (ThomasTNO) [reply docs/configuring.rst]: Added a link to `gtag.js` in the docs

**Review by Blendify — APPROVED** [review]

**Comment by ThomasTNO** [comment]
Will this be taken into account for the next release?

**Pull request merged**